docker/fixtures/index.ts

```diff
@@ -1,82 +1,59 @@
+import { toError } from "fp-ts/lib/Either";
+import { TaskEither, tryCatch } from "fp-ts/lib/TaskEither";
 import {
-  CollectionMeta,
-  DocumentClient as DocumentDBClient,
-  UriFactory
-} from "documentdb";
-import { Either, left, right } from "fp-ts/lib/Either";
-import {
-  Profile,
+  NewProfile,
   PROFILE_COLLECTION_NAME,
   ProfileModel
 } from "io-functions-commons/dist/src/models/profile";
 import {
-  Service,
+  NewService,
   SERVICE_COLLECTION_NAME,
   ServiceModel
 } from "io-functions-commons/dist/src/models/service";
-import * as documentDbUtils from "io-functions-commons/dist/src/utils/documentdb";
-import { getRequiredStringEnv } from "io-functions-commons/dist/src/utils/env";
+import { getConfigOrThrow } from "../../utils/config";
+import { cosmosdbClient } from "../../utils/cosmosdb";
 
-const cosmosDbKey = getRequiredStringEnv("CUSTOMCONNSTR_COSMOSDB_KEY");
-const cosmosDbUri = getRequiredStringEnv("CUSTOMCONNSTR_COSMOSDB_URI");
-const cosmosDbName = getRequiredStringEnv("COSMOSDB_NAME");
+const config = getConfigOrThrow();
+const cosmosDbName = config.COSMOSDB_NAME;
 
-const documentDbDatabaseUrl = documentDbUtils.getDatabaseUri(cosmosDbName);
-
-const documentClient = new DocumentDBClient(cosmosDbUri, {
-  masterKey: cosmosDbKey
-});
-
-function createDatabase(databaseName: string): Promise<Either<Error, void>> {
-  return new Promise(resolve => {
-    documentClient.createDatabase({ id: databaseName }, (err, _) => {
-      if (err) {
-        return resolve(left<Error, void>(new Error(err.body)));
-      }
-      resolve(right<Error, void>(void 0));
-    });
-  });
+function createDatabase(databaseName: string): TaskEither<Error, void> {
+  return tryCatch(
+    () => cosmosdbClient.databases.createIfNotExists({ id: databaseName }),
+    toError
+  ).map(() => void 0);
 }
 
 function createCollection(
+  databaseName: string,
   collectionName: string,
   partitionKey: string
-): Promise<Either<Error, CollectionMeta>> {
-  return new Promise(resolve => {
-    const dbUri = UriFactory.createDatabaseUri(cosmosDbName);
-    documentClient.createCollection(
-      dbUri,
-      {
-        id: collectionName,
-        partitionKey: {
-          kind: "Hash",
-          paths: [`/${partitionKey}`]
-        }
-      },
-      (err, ret) => {
-        if (err) {
-          return resolve(left<Error, CollectionMeta>(new Error(err.body)));
-        }
-        resolve(right<Error, CollectionMeta>(ret));
-      }
-    );
-  });
+): TaskEither<Error, ContainerResponse> {
+  return tryCatch(
+    () =>
+      cosmosdbClient
+        .database(databaseName)
+        .containers.createIfNotExists({ id: collectionName, partitionKey }),
+    toError
+  );
 }
 
-const servicesCollectionUrl = documentDbUtils.getCollectionUri(
-  documentDbDatabaseUrl,
-  SERVICE_COLLECTION_NAME
-);
-const serviceModel = new ServiceModel(documentClient, servicesCollectionUrl);
+const servicesContainer = cosmosdbClient
+  .database(config.COSMOSDB_NAME)
```

Comment:
we should remove the whole folder fixtures since fixtures are now handled with its own project